commit 25e6a02b412aec1c5d9cbd5d7d239b6bd2cde235 (refs/changes/54/5454/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-30T19:00:03+00:00 (8 months ago)
    
    MANTA-4093 nginx should not modify its source tree during the build
    Reviewed By: Robert Mustacchi <rm@joyent.com>

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 00000000..9544294d
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,2 @@
+/Makefile
+/objs
